resources/prosody-plugins/mod_fmuc.lua

```diff
@@ -236,3 +239,44 @@ end
 process_host_module(main_domain, function(host_module, host)
     host_module:hook('iq/host', stanza_handler, 10);
 end);
+
+-- only live chat is supported for visitors
+module:hook("muc-occupant-groupchat", function(event)
```

Comment:
Do we have anything that blocks private messages sent by visitors?

Reply:
Nope, just UI is hidden. I will take a look.